mbuild/compound.py

```diff
@@ -1304,16 +1304,18 @@ def save(self, filename, show_ports=False, forcefield_name=None,
         # pad non periodic faces with .25nm buffers
         if box is None:
             box = self.boundingbox
+            box_vec_max = box.maxs.tolist()
+            box_vec_min = box.maxs.tolist()
```

Comment:
This should definitely be `box.mins` but could you explain why the list cast in necessary here?